docs/docs/cmd/aad/app/app-role-remove.md

```diff
@@ -1,70 +1,76 @@
-# aad app role delete
```

Comment:
After renaming the file, we should update the link to it in release notes. Right now we're getting an error when building docs